helpdesk/overview/sla.rst

```diff
@@ -0,0 +1,45 @@
+==============================
+Service Level Agreements (SLA)
+==============================
+
+Service Level Agreements (SLA) are commitments you make with your customers to outline how a
+service is delivered. It bolsters trust between you and your customers as it makes clear what
+needs to be done, to what standard, and when.
+
+Create your policies
+====================
+
+| First, enable the feature on the settings of the team you would like policies to be applied,
+  going to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams`.
+| Create your policies through the team’s settings page, or go to :menuselection:`Helpdesk -->
+  Configuration --> SLA Policies`.
+
+.. image:: media/new_sla.png
+   :align: center
+   :height: 340
+   :alt: View of an SLA form in Odoo Helpdesk
+
+| Choose to which **Team** the policy is relevant, and the minimum priority a ticket needs to have
+  for the policy to be applied.
```

Comment:
```suggestion
| Choose to which **Team** the policy is relevant and the minimum priority a ticket needs to have
  for the policy to be applied.
```

Reply:
hum.. I will keep the comma because, otherwise, the sentence gets tiring and hard to be read.

Reply:
I still think that it is grammatically better to remove the comma, but I get your point.

Reply:
ok.. changed!